addons/web/static/src/views/fields/many2many_tags/many2many_tags_field.js

```diff
@@ -199,6 +199,17 @@ export class Many2ManyTagsField extends Component {
             getFieldDomain(this.props.record, this.props.name, this.props.domain),
         ]).toList(this.props.context);
     }
+
+    mapRecord(record) {
+        const records = this.props.record.data[this.props.name].records;
+        const isSelected = records.some((r) => r.resId === record.id);
+        return {
+            value: record.id,
+            label: record.display_name ? record.display_name.split("\n")[0] : _t("Unnamed"),
+            displayName: record.display_name,
+            classList: isSelected ? "dropdown-item-selected" : "",
```

Comment:
```suggestion
            classList: isSelected ? "fw-bold" : "",
```

Reply:
That does not work, it's obviously the way to do but the css class is added on the `li.o-autocomplete--dropdown-item`.
This `li` contains a `a.dropdown-item` and the css class `dropdown-item` sets a font weight of 400 which override the `fw-bold` and never make the text bold.